webapp/sources/rudder/rudder-rest/src/test/scala/com/normation/rudder/rest/TraitTestApiFromYamlFiles.scala

```diff
@@ -137,35 +132,22 @@ object TraitTestApiFromYamlFiles {
     (rudderApi, liftRules)
   }
 
-}
-
-trait TraitTestApiFromYamlFiles extends Specification with BoxSpecMatcher with JsonSpecMatcher {
-
-  // source directory for yaml file, must be in the test class path (accessed as a resource).
-  // For example, if yaml test files are under src/test/resources/api, then just use "api" here
-  def yamlSourceDirectory:  String
-  // the base temp directory where are stored yaml test files
-  // It can have a `templates` subdirectory where templates for transformation are store
-  // If you have template, that directory needs to be writable to allows to copy processed template there.
-  // You need to take care of the cleanup by yourself.
-  def yamlDestTmpDirectory: File
-  // transformation are pre-processing of template before running the test, typically to set api version, random things, etc
-  def transformations:      Map[String, String => String]
-
-  // the liftRules to use for API
-  def liftRules: LiftRules
```

Comment:
Since we don't have trait anymore, these things are passed as additionnal function parameters